src/chrome/content/rules/pronoun.is.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid certificate:
+		hubots.pronoun.is
+		my.pronoun.is
+		www.pronoun.is
```

Comment:
It seems that this host has a wildcard DNS. Could you add a comment about this?

Since `my` and `www` serve the same content as `^` we can secure them by redirecting to `^`.